accounts/tests.py

```diff
@@ -250,6 +266,25 @@ def test_lab_researcher_can_create_study(self):
     def test_unaffiliated_researcher_cannot_create_study(self):
         self.assertFalse(self.unaffiliated_researcher.can_create_study())
 
+    def test_create_user_lowercases_username(self):
+        # TODO: Do we actually use `create_user` anywhere?
```

Comment:
only in create_superuser - otherwise users are created via the ParticipantSignupForm & ResearcherRegistrationForm. Ok for me to push addition of clean_username methods to those on this branch?

Reply:
Which branch are those currently on?